src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -1451,6 +1451,10 @@ private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, string srcVariable, stri
             void ISaveAsOnnx.SaveAsOnnx(OnnxContext ctx)
             {
                 Host.CheckValue(ctx, nameof(ctx));
+
+                const int minimumOpSetVersion = 11;
+                ctx.CheckOpSetVersion(minimumOpSetVersion, LoaderSignature);
```

Comment:
Can we move this checks closer to where the operators are actually used?

Reply:
Sure, will move it to SaveAsOnnxCore() and right above those op usage